commit daba2b3313077c3be9b424d8bcba2d258d7048d3
Author: nmittler <nathanmittler@google.com>
Date:   Wed Feb 11 09:06:09 2015 -0800

    Removing debugging change from unit test.
    
    Motivation:
    
    HttpToHttp2ConnectionHandlerTest was accidentally modified with a
    debugging value for WAIT_TIME_SECONDS.
    
    Modifications:
    
    Reverted the change.
    
    Result:
    
    original wait time restored.

diff --git a/codec-http2/src/test/java/io/netty/handler/codec/http2/HttpToHttp2ConnectionHandlerTest.java b/codec-http2/src/test/java/io/netty/handler/codec/http2/HttpToHttp2ConnectionHandlerTest.java
index 8584998..04d5f7a 100644
--- a/codec-http2/src/test/java/io/netty/handler/codec/http2/HttpToHttp2ConnectionHandlerTest.java
+++ b/codec-http2/src/test/java/io/netty/handler/codec/http2/HttpToHttp2ConnectionHandlerTest.java
@@ -70,7 +70,7 @@ import org.mockito.stubbing.Answer;
  * Testing the {@link HttpToHttp2ConnectionHandler} for {@link FullHttpRequest} objects into HTTP/2 frames
  */
 public class HttpToHttp2ConnectionHandlerTest {
-    private static final int WAIT_TIME_SECONDS = 500;
+    private static final int WAIT_TIME_SECONDS = 5;
 
     @Mock
     private Http2FrameListener clientListener;
